client/my-sites/ads/form-earnings.jsx

```diff
@@ -81,198 +71,229 @@ module.exports = React.createClass( {
 			showEarningsNotice: false,
 			showWordadsInfo: false,
 			showSponsoredInfo: false,
-			showAdjustmentInfo: false
+			showAdjustmentInfo: false,
 		} );
-	},
+	}
 
-	_fetchIfEmpty: function( site ) {
+	fetchIfEmpty( site ) {
 		site = site || this.props.site;
 		if ( ! site || ! site.ID ) {
 			return;
 		}
 
 		if ( EarningsStore.getById( site.ID ).earnings ) {
-			debug( 'initial fetch not necessary' );
 			return;
 		}
 
 		// defer fetch requests to avoid dispatcher conflicts
-		setTimeout( function() {
-			WordadsActions.fetchEarnings( site )
+		setTimeout( () => {
+			WordadsActions.fetchEarnings( site );
 		}, 0 );
-	},
+	}
 
-	toggleEarningsNotice: function( event ) {
+	updateSettings = () => {
+		this.setState( this.getSettingsFromStore() );
+	};
+
+	handleEarningsNoticeToggle = ( event ) => {
```

Comment:
Maybe I'm going overboard, but this is pretty much identical to `handleInfoToggle`. One of them could easily be removed.